lib/msal-node/src/config/ManagedIdentityId.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    DEFAULT_MANAGED_IDENTITY_ID,
+    ManagedIdentityIdType,
+} from "../utils/Constants";
+
+export class ManagedIdentityId {
+    private _id: string;
```

Comment:
inconsistent naming. Pls don't follow the .NET naming conventions. There is a linter in MSAL JS, pls ensure it runs and passes.

Reply:
I don't understand what you're saying. I run the linter before every push to GitHub.

Reply:
If I look at existing files like `ClientCredentialClient.ts` I see that private variables are defined as 

`private scopeSet: ScopeSet;`

But in your case you use 

`private _id: string`

So the naming is inconsistent. Maybe the linter doesn't work or doesn't pick up on this? In any case, pls use consistent naming of vars.